sophia/src/term.rs

```diff
@@ -275,21 +290,11 @@ where
     }
 
     /// Return a typed literal term.
-    ///
-    /// # Panics
-    /// Panics if `dt` is not an IRI.
     pub fn new_literal_dt_unchecked<U>(txt: U, dt: Term<T>) -> Term<T>
     where
-        T: From<U> + Debug,
+        T: From<U>,
     {
-        if let Iri(dt) = dt {
-            Literal(T::from(txt), Datatype(dt))
-        } else {
-            panic!(format!(
-                "new_literal_dt_unchecked expects Term::Iri as dt, got {:?}",
-                dt
-            ))
-        }
+        Literal(T::from(txt), Datatype(dt.try_into().unwrap()))
```

Comment:
The `unwrap()` in this line may panic, so why removing the related section in the documentation?
```
    ///
    /// # Panics
    /// Panics if `dt` is not an IRI.
```

Reply:
Your right, re-included documentation (don't know why I removed it in the first place...)